doc/source/cluster/kubernetes/examples/rayjob-batch-inference-example.md

```diff
@@ -86,31 +84,13 @@ Sample output:
 [...]
 Status:
   Dashboard URL:          rayjob-sample-raycluster-j6t8n-head-svc.default.svc.cluster.local:8265
-  End Time:               2023-08-22T22:48:35Z
-  Job Deployment Status:  Running
+  End Time:               ...
+  Job Deployment Status:  Complete
```

Comment:
In the master branch, if JobStatus is SUCCEEDED, the JobDeploymentStatus will eventually transition to Complete.